lib/model.js

```diff
@@ -1353,7 +1356,10 @@ Model.prototype.findAll = function(options) {
   tableNames[this.getTableName(options)] = true;
   options = optClone(options);
 
-  _.defaults(options, { hooks: true });
+  _.defaults(options, { hooks: true, kenophobic: false });
```

Comment:
Just tested, I didn't knew `defaults` behavior, but it will only overwrite `undefined` values, even if value is `false` it will not assign the `true` value from model options.